------------------------------------------------------------------------

I am a genomics scientist working with large-scale bioinformatic and
genetic datasets.

------------------------------------------------------------------------

I’m an experienced bioinformatician, geneticist and molecular biologist.

My main tools for programming and data analysis are Python and R. I use
workflow managers, version control, containerisation and
high-performance computing for bioinformatics. I’m also experienced with
bash scripting and \*nix system administration.

I am currently using my strategic development and user support skills in
the context of a team working to provide national bioinformatics
infrustructure, in collaboration with a distributed global team.

I am skilled at the entire wet-lab workflow for producing short and long
read sequencing libraries, from nucleic acid extraction to library
preparation and running instruments for 2<sup>nd</sup>- and
3<sup>rd</sup>-generation sequencing. I have experience with classical
genetic approaches with live organisms, including transformation,
crosses, microdissection and bioassays.

I enjoy communicating complex bioinformatics and genetic results to
research partners, external stakeholders and students. I visualise and
share my results with ggplot2, Markdown, Pandoc, TeX and a bit of html &
css. I teach complex bioinformatics algorithms to postgraduate students
in classroom and one-on-one settings.

<!-- I use git, Snakemake, Singularity and Docker to keep my work reproducible.
I've used job schedulers like SLURM, LSF and SGE for managing jobs in high-performance computing environments.
I'm comfortable working with, installing and administrating *nix operating systems (Linux, macOS and FreeBSD).
 -->
<!-- Collaboration -->

I’m interested in research related to agriculture and sustainable
development, tools and techniques in bioinformatics and evolutionary and
comparative biology.

