---
# title: "Thomas Harrop"
layout: single
author_profile: true
toc: true
toc_label: "About me"
toc_sticky: true
# toc_icon: "fas fa-bars"
---
------------------------------------------------------------------------

I work with large-scale genetic, genomic and transcriptomic datasets.  
I use `R` and `python` to tidy, analyse and visualise data and
communicate results.  
I generate data using 2<sup>nd</sup>- and 3<sup>rd</sup>-generation
sequencing.

------------------------------------------------------------------------

About me {#about-me}
--------

### Technical skills {#technical-skills}

My main tools for programming and data analysis are `python` and `R` .
I’m also experienced with `bash` scripting. I visualise and communicate
my results with `ggplot2`, `markdown`, `pandoc`, `TeX` and a bit of
`html` & `css`. I use `git`, `snakemake`, `singularity` and `Docker` to
keep my work reproducible. I’ve used workflow managers (`SLURM`, `LSF`
and `SGE`) for managing jobs in high-performance computing environments.
I’m comfortable working with, installing and administrating \*nix
operating systems (`Linux`, `macOS` and `FreeBSD`).

I’m an experienced molecular biologist. I am skilled at the entire
wet-lab workflow for producing short and long read sequencing libraries,
from nucleic acid extraction to library preparation and running
sequencing instruments. I have experience with classical genetic
approaches with live organisms, including transformation, crosses,
microdissection and bioassays.

<!-- Collaboration -->

### Research {#research}

I’m interested in research related to agriculture and development, tools
and techniques in bioinformatics and evolutionary and comparative
biology.

### Personal information {#personal-information}

Languages
:   English (native)  
    German (TELC Level B2)  
    French (TELC Level B1)

Nationalities
:   Australian and British

Experience {#experience}
----------

### Research positions {#research-positions}

2017–Present

:   **Post-doctoral researcher**, Laboratory for Evolution and
    Development, Biochemistry Department, University of Otago, Dunedin,
    New Zealand.  
    I am currently working on a project to develop and apply tools to
    integrate genomic selection to breeding programmes to improve
    honeybee performance.

2016–2017

:   **Post-doctoral researcher**, Laboratory for Evolution and
    Development, Biochemistry Department, University of Otago, Dunedin,
    New Zealand.  
    In this project, I worked on a bioprotection project investigating
    the resistance of Argentine stem weevil to its biological control
    agent.

2014–2016
:   **Post-doctoral researcher**, UMR DIADE, Institut de Recherche pour
    le Développement (IRD), Montpellier, France.  
    I used machine learning, data mining, statistical analyses and other
    bioinformatic techniques to study the development of inflorescence
    architecture in rice, which affects crop yield. We used laser
    microdissection and RNA sequencing to characterise molecular events
    that occur in the developing inflorescence, and applied these
    results to study the effects of selection on gene expression during
    the parallel domestications of African and Asian rice.

2013–2014
:   **Post-doctoral researcher**, Department of Plant Breeding and
    Genetics, Max Planck Institute for Plant Breeding Research, Cologne,
    Germany.  
    At the Max Planck Institute I worked on a bioinformatic and
    molecular biology project to compare the transcriptional response of
    different land plants to UV-B exposure, which acts as an
    environmental signal.

### Professional positions {#professional-positions}

2004–2007
:   **Medical scientist**, Network Pathology, the Austin Hospital,
    Heidelberg, Australia.

2001–2004
:   **Laboratory assistant**, Network Pathology, the Austin Hospital,
    Heidelberg, Australia.

Education {#education}
---------

2006–2012
:   **Doctor of Philosophy**, Department of Genetics, the University of
    Melbourne, Parkville, Australia.  
    Thesis: [*The functions of cytochrome P450s in
    Drosophila*.](http://cat.lib.unimelb.edu.au/record=b4827945~S32)  
    During my Ph.D. I used classical genetics and modern molecular
    biology, including transgenic manipulation of gene expression and
    comparative studies of gene function, to investigate the functions
    of cytochrome P450 genes in the Drosophila genus.

2002–2005
:   **Bachelor of Science with Honours (first class)**, the University
    of Melbourne, Parkville, Australia.

Awards
:   **2006–2009**, Australian Postgraduate Award

### Teaching {#teaching}

2007–2011
:   **3<sup>rd</sup> Year Genetics**, Department of Genetics, The
    University of Melbourne, Parkville, Australia.

2006–2009
:   **High School Biology**, Department of Genetics, The University of
    Melbourne, Parkville, Australia.

2006–2008
:   **High School Biology**, Gene Technology Access Centre, The
    University High School, Parkville, Australia.

2005–2008
:   **1<sup>st</sup> Year Genetics**, The University of Melbourne,
    Parkville, Australia.

### Students co-supervised {#students-co-supervised}

Marissa Le Lec
:   **Bachelor of Science with Honours**, Genetics, University of Otago,
    Dunedin, New Zealand.

Swarit Jaisal
:   **Master of Science**, Life Science Informatics, Universität Bonn,
    Germany.

Axel Verdier
:   **Master of Science**, Sciences et Technologies de l’Information et
    de la Communication pour la Santé, Université de Montpellier,
    France.

Publications {#references}
------------

### Peer-reviewed journals {#peer-reviewed-journals}

Morgan, B.M., Brown, A.N., Deo, N., **Harrop, T.W.R.**, Taiaroa, G.,
Mace, P.D., Wilbanks, S.M., Merriman, T.R., Williams, M.J.A., McCormick,
S.P.A. (2020). Nonsynonymous SNPs in *LPA* homologous to plasminogen
deficiency mutants represent novel null apo(a) alleles. Journal of Lipid
Research 61(3), 432–44.
[doi:10.1194/jlr.M094540](https://doi.org/10.1194/jlr.M094540).

Lamichhane, R., Schneider, M., de la Harpe, S.M., **Harrop, T.W.R.**,
Hannaway, R.F., Dearden, P.K., Kirman, J.R., Tyndall, J.D.A., Vernall,
A.J., and Ussher, J.E. (2019). TCR- or Cytokine-Activated CD8+
Mucosal-Associated Invariant T Cells Are Rapid Polyfunctional Effectors
That Can Coordinate Immune Responses. Cell Reports 28, 3061-3076.e5.
[doi:10.1016/j.celrep.2019.08.054](https://doi.org/10.1016/j.celrep.2019.08.054).

**Harrop, T.W.R.**, Mantegazza, O., Luong, A.M., Béthune, K., Lorieux,
M., Jouannic, S., and Adam, H. (2019). A set of *AP2*-like genes is
associated with inflorescence branching and architecture in domesticated
rice. Journal of Experimental Botany 70, 5617–5629.
[doi:10.1093/jxb/erz340](https://doi.org/10.1093/jxb/erz340).

Guerrero, F.D., Bendele, K.G., Ghaffari, N., Guhlin, J., Gedye, K.R.,
Lawrence, K.E., Dearden, P.K., **Harrop, T.W.R.**, Heath, A.C.G., Lun,
Y., et al. (2019). The Pacific Biosciences *de novo* assembled genome
dataset from a parthenogenetic New Zealand wild population of the
longhorned tick, *Haemaphysalis longicornis* Neumann, 1901. Data in
Brief 27, 104602.
[doi:10.1016/j.dib.2019.104602](https://doi.org/10.1016/j.dib.2019.104602).

**Harrop, T.W.**<sup>§</sup>, Denecke, S.<sup>§</sup>, Yang, Y.T., Chan,
J., Daborn, P.J., Perry, T., Batterham, P., (2018). Evidence for
activation of nitenpyram by a mitochondrial cytochrome P450 in
*Drosophila melanogaster*. Pest Management Science 74, 1616–1622.
[doi:10.1002/ps.4852](https://doi.org/10.1002/ps.4852).

Ta, K. N., Adam. H., Staedler, Y.M., Schönenberger, J., **Harrop.
T.W.R.**, Tregear, J., Do, N.V.,Gantet, P., Ghesquière, A., and
Jouannic, S. (2017). Differences in meristem size and expression of
branching genes are associated with variation in panicle phenotype in
wild and domesticated African rice. EvoDevo 8: 2.
[doi:10.1186/s13227-017-0065-y](http://dx.doi.org/10.1186/s13227-017-0065-y).

**Harrop, T.W.R.**<sup>§</sup>, Ud Din, I.<sup>§</sup>, Gregis, V.,
Osnato, M., Jouannic, S., Adam, H. and Kater, M. (2016). Gene expression
profiling of reproductive meristem types in early rice inflorescences by
laser microdissection. The Plant Journal 86, 75–88.
[doi:10.1111/tpj.13147](http://dx.doi.org/10.1111/tpj.13147).

**Harrop, T.W.R.**<sup>§</sup>, Pearce, S.L.<sup>§</sup>, Daborn, P.J.
and Batterham, P. (2014). Whole-genome expression analysis in the third
instar larval midgut of *Drosophila melanogaster*. G3 (Bethesda) 4,
2197–2205.
[doi:10.1534/g3.114.013870](http://g3journal.org/cgi/doi/10.1534/g3.114.013870).

**Harrop, T.W.R.**, Sztal, T., Lumb, C., Good, R.T., Daborn, P.J.,
Batterham, P. and Chung, H. (2014). Evolutionary changes in gene
expression, coding sequence and copy-number at the *Cyp6g1* locus
contribute to resistance to multiple insecticides in Drosophila. Plos
One 9, e84879.
[doi:10.1371/journal.pone.0084879](http://dx.plos.org/10.1371/journal.pone.0084879).

Daborn, P.J., Lumb, C., **Harrop, T.W.R.**, Blasetti, A., Pasricha, S.,
Morin, S., Mitchell, S.N., Donnelly, M.J., Müller, P. and Batterham, P.
(2012). Using *Drosophila melanogaster* to validate metabolism-based
insecticide resistance from insect pests. Insect Biochemistry and
Molecular Biology 42, 918–924.
[doi:10.1016/j.ibmb.2012.09.003](http://linkinghub.elsevier.com/retrieve/pii/S0965174812001269).

<sup>§</sup> Equal contribution

### Book chapters {#book-chapters}

Cridge, A., **Harrop, T. W. R.**, Lovegrove, M., Remnant, E. & Dearden,
P. (2017). *Nutrition and Epigenetic Change in Insects: Evidence and
Implications*, in Advances in Insect Physiology (ed. Verlinden, H.) 53,
31–54 (Academic Press).
[doi:10.1016/bs.aiip.2017.06.001](http://www.sciencedirect.com/science/article/pii/S006528061730022X)

### Selected scientific presentations {#selected-scientific-presentations}

*The genomes of three invasive social wasps*. **Oral presentation** at
the Annual Conference of the Genetics Society of Australasia (GSA),
Melbourne, Australia, 2019. **Symposium chair**: Genomics &
bioinformatics.

*Comparative genomics of sexual and asexual parasitoid wasps from the
Microctonus genus*. **Poster presentation** at the 7<sup>th</sup>
meeting of the European Society for Evolutionary Developmental Biology,
Galway, Ireland, 2018.

*Genomics of a weevil pest and its parasitoid biocontrol agent*.
**Poster presentation** at the Annual Conference of the Genetics Society
of Australasia (GSA) with the New Zealand Society for Biochemistry and
Molecular Biology (NZSBMB), Dunedin, New Zealand, 2017.

De novo *sequencing of Argentine stem weevil and its parasitoid wasp
biocontrol agent*. **Poster presentation** at the 10<sup>th</sup> Annual
Arthropod Genomics Symposium, Notre Dame University, South Bend, U.S.A.,
2017.

*Sequencing and community annotation of the* Vespula vulgaris *genome*.
**Oral presentation** at the Entomological Society of NZ Conference,
Wellington, New Zealand, 2017.

*Genes controlling transition of reproductive meristem identity during
inflorescence development*. **Oral presentation** at the 14<sup>th</sup>
International Symposium on Rice Functional Genomics, Montpellier,
France, 2016.

*Mechanisms underlying the parallel evolution of inflorescence phenotype
during independent domestication of African and Asian rice*.
**Contributed oral presentation** at the 6<sup>th</sup> meeting of the
European Society for Evolutionary Developmental Biology, Uppsala,
Sweden, 2016. **Joint symposium chair**: Branching across the tree of
life.

*Gene expression during* O. sativa *panicle development*. **Invited
seminar** at the Agricultural Genetics Institute, Hanoi, Vietnam, 2015.

*Molecular mechanisms underlying the phenotypic convergence of
inflorescence architecture in domesticated rice species*. **Poster
presentation** at the Society for Molecular Biology and Evolution annual
meeting, Vienna, Austria, 2015.

*Transcriptomic analysis of early developmental stages of the rice
panicle*. **Poster presentation** at the Workshop in Molecular
Mechanisms Controlling Flowering, Aiguablava, Spain, 2015

*RNA interference of the Cytochrome P450 redox partners* Cpr *and* dare
*uncovers novel P450 functions in* Drosophila melanogaster. **Poster
presentation** at the 50<sup>th</sup> Annual Drosophila Research
Conference, Chicago, USA, 2009.

------------------------------------------------------------------------
